### PR TITLE
v2-ui-global-settings

### PR DESCRIPTION
**Pull request by harishshg**
Fixes # .

Changes proposed in this pull request:
- global settings list grid 
- edit global settings

@deepfence/engineering

### CODE REVIEW

File: deepfence_frontend/apps/dashboard/src/features/settings/pages/EditGlobalSettings.tsx [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+  fieldErrors?: {
+    value?: string;
+  };
+};
+
+export const action: ActionFunction = async ({
+  request,
+}): Promise<UpdateActionReturnType> => {
+  const formData = await request.formData();
+  // add console_url which is the origin of request
```
Comment (manV) [inline deepfence_frontend/apps/dashboard/src/features/settings/pages/EditGlobalSettings.tsx:39]: why is this needed?

**Comment by milan-deepfence** [comment]
Won't it be better to have edit on either Modal or have in a side panel? This may help :
1. not to call getSetting api
2. Remove Back button(back button inside tab is ugly)

Screenshot 2023-04-17 at 9 43 23 PM

File: deepfence_frontend/apps/dashboard/src/features/settings/pages/EditGlobalSettings.tsx (hunk 1 above)
Comment (harishshg) [reply deepfence_frontend/apps/dashboard/src/features/settings/pages/EditGlobalSettings.tsx]: fixed
